dcc-portal-ui/app/scripts/share/views/share.icon.html

```diff
@@ -1 +1,7 @@
-<a data-ng-click="shareCtrl.getShortUrl(shareParams, true)"><i class="icon-share"></i></a>
\ No newline at end of file
+<a 
+    popover-title="Share Link"
+    popover-template = "'/scripts/share/views/share.popover.html'" 
+    popover-placement = "bottom"
+    popover-is-open = "shareCtrl.popoverIsOpen"
+    data-ng-click = "shareCtrl.handleClickShareButton()"
```

Comment:
Pass in `shareParams` to `handleClickShareButton()`

Log out the contents to see what shape that object has